app/serializers/prison_transfer_reason_serializer.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
-class PrisonTransferReasonSerializer < ActiveModel::Serializer
+class PrisonTransferReasonSerializer
+  include JSONAPI::Serializer
+
+  set_type :prison_transfer_reasons
+
   attributes :id, :title, :key, :disabled_at
```

Comment:
More with the id in the attributes 🤦

Reply:
Yep, but note this example is unchanged so has been around for a while. I do agree it's likely not needed so will aim to clean these all up.